lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -412,10 +412,23 @@ export abstract class ClientApplication {
             : this.config.auth.azureCloudOptions;
 
         // using null assertion operator as we ensure that all config values have default values in buildConfiguration()
-        this.logger.verbose(
-            `building oauth client configuration with the authority: ${authority}`,
-            requestCorrelationId
-        );
+        this.logger.verbose("here-----------------------------------------------------------------------------------------------------------------");
+
+        if(azureRegionConfiguration?.azureRegion)
```

Comment:
Shouldn't this logging block go after line 432, where you do discovery. This way you can output "msal was configured with authority X and region Y and it uses the token endpoint Z"